docs/user_guide/1_setting_up_the_hand.md

```diff
@@ -99,84 +100,41 @@ see all joints of the hand move slightly on power up or reset or power up.
 
-  Before setting up the docker container, the EtherCAT interface ID for the hand needs to be discovered. In order to do so, after plugging the hand’s ethernet cable into your machine and powering it up, please run
-
-  ```bash
-  $ sudo dmesg
-  ```
-  command in the console. At the bottom, there will be information similar to the one below:
-
-  ```bash
-  [490.757853] IPv6: ADDRCONF(NETDEV_CHANGE): enp0s25: link becomes ready
-  ```
-  In the above example, ‘enp0s25’ is the interface ID that is needed.
+#### On a new PC using the one-liner
+We have created a one-liner that is able to install Docker, download the docker image and create a new container for you. It will also create desktop icons, one to start the container, one launch the hand driver on the control box and one to save the log files locally. To use it, you first need to have a PC with Ubuntu installed on it (preferable version 16.04), then follow these steps:
 
 * **Get ROS Upload login credentials**
 
   If you want to upload technical logged data (ROS logs, backtraces, crash dumps etc.) to our server and notify the Shadow's software team to investigate your bug then you need to enable logs uploading in the one-liner. In order to use this option you need to obtain a unique upload key by emailing sysadmin@shadowrobot.com. When you receive the key you can use it when running the one-liner installation tool. To enable the logs uploading you need to add the command line option ```use_aws=true``` to the one-liner.
   After executing the one-liner, it will prompt you to enter your upload key and press enter to continue. Please copy and paste your key from the email you received by Shadow Robot.
 
-* **Check your hand configuration branch**:
-
-  You should have the name of your [sr_config](https://github.com/shadow-robot/sr-config) hand branch which contains the specific configuration of your hand (calibration, controller tuning etc…).
-  Usually it is something like this: ``shadowrobot_XXXXXX``. Where XXXXXX are the 6 digits contained in the serial number of the hand labelled underneath the robot base.
-
-  If you are unsure please contact us.
-
 * **Run the one-liner**:
 
-  The one-liner will install Docker, pull the image from Docker Hub, and create and run a container with the parameters specified. In order to use it, use the following command:
-
-  **Please remember to replace [EtherCAT interface ID] with your Interface ID and [sr_config_branch] with your unique sr_config branch**
+  The one-liner will install Docker, pull the image from Docker Hub, and create and run a container with the parameters specified. In order to use it, run the following command:
 
   ROS Kinetic (Recommended):
   ```bash
-  $ bash <(curl -Ls bit.ly/run-aurora) docker_deploy product=hand_e ethercat_interface=[EtherCAT interface ID] config_branch=[sr_config_branch]
+  $ bash <(curl -Ls bit.ly/run-aurora) server_and_nuc_deploy --read-input docker_username --read-secure docker_password --limit 'server'
```

Comment:
don't use the --limit 'server' because then the dhcp playbook won't run and certain NUC tasks won't happen like ssh keys, and setting the correct ROS_MASTER_URI on the NUC (which depends on the hostname of the server laptop)

Reply:
Instead, use this:
bash <(curl -Ls bit.ly/run-aurora) server_and_nuc_deploy --read-secure sudo_password,customer_key ethercat_interface=enx000ec6511588 config_branch=shadowrobot_200117 product=hand_e reinstall=true use_aws=true hand_serial=2378 internet_interface_name=enp8s0f1 dhcp_interface_name=enx000ec653b3bc dhcp_server_mac="00:0e:c6:53:b3:bc" dhcp_client_mac="00:0e:c6:53:b4:35" upgrade_check=true launch_hand=true

Reply:
In the future, when I make running aurora simpler, the above can be reduced to this: 
bash <(curl -Ls bit.ly/run-aurora) server_and_nuc_deploy --read-secure sudo_password,customer_key ethercat_interface=enx000ec6511588 config_branch=shadowrobot_200117 product=hand_e reinstall=true use_aws=true upgrade_check=true launch_hand=true